docs/source/model_design/risk_factors.rst

```diff
@@ -48,8 +48,111 @@ Definitions
+3. Measure of prevalence: 20.7% of women attending antenatal care at rural clinics Siaya county, western Kenya were HIV positive in 2015
+
+Incidence
+^^^^^^^^^
+Incidence focuses on new cases. There are two measures of incidence: risk and rate. 
+
+*Incidence risk*, also called *incidence proportion*, *attack rate*, or *cumulative incidence* is the probability of occurence of disease among a disease free, at risk, population during a specified time period. It is the number of new cases of disease during a defined period of time divided by the population at the start of the time period. It is not interpretable without specification of the time period to which it applies. The counterfactual of the incidence proportion is the survival proportion which is 1-incidence proportion.                                                 
+
+*Incidence rate* has the same numerator as incidence risk, that is the appearance of new cases. In contrast to risks, which relate the number of new cases to the size of the population at risk in the beginning of the period studied, rates relate the number of new cases to the person-time (Y) at risk, a measure that takes into account changes in the size of the population at risk during the follow-up period. The rate takes into account the fact that some people who start at risk do not remain at risk during the whole period, because they develop the disease, or die, or leave the population by migrating, refusing to continue to participate in the study etc. Others may join the population at risk after the beginning of the period, through birth, migration into the area, recruitment into the study, etc. The denominator in a rate (Y) is thus the sum of the time each person in the study population remained at risk during the study period. This is called the person-time experience at risk, and is expressed in units of person-time: person-years at risk, person-days at risk, baby-weeks at risk etc.
+
+For rare diseases, risk and rates are numerically similar. 
+
+Prevalence
+^^^^^^^^^^
+Prevalence focuses on existing states. Prevalence of a state (such as the 'with condition state') at a point in time may be defined as the proportion of a population in that state at that time; thus prevalence is the proportion of persons in a defined population that have the outcome under study in a defined period of time. 
+
+*Point prevalence* is the number of current cases (new and pre-existing) at a defined instant in time. The denominator is the population at the same defined instant in time. Eg. the percentage of people with schistosomiasis parasites in the blood in a village in Kenya in a survey in December 2019; the proportion of people who have diabetes in China today. 
+
+*Period prevalence* is the number of current cases (new and pre-existing) over a defined period of time. The denominator is the average or mid-period population. 
```

Comment:
Your wording for the definition of prevalence seems incorrect. In particular, you say prevalence is "the _number_ of current cases," but I'm pretty sure prevalence should be a _proportion_, not a count of cases. I've heard the _number_ of cases referred to as "prevalent cases."  Perhaps you were referring to what to use in the numerator to calculate prevalence?

These are the definitions of prevalence I found on the [CDC website](https://www.cdc.gov/csels/dsepd/ss1978/lesson3/section2.html):

> Prevalence, sometimes referred to as **prevalence rate**, is the proportion of persons in a population who have a particular disease or attribute at a specified point in time or over a specified period of time.
>
> **Point prevalence** refers to the prevalence measured at a particular point in time. It is the proportion of persons with a particular disease or attribute on a particular date.
>
> **Period prevalence** refers to prevalence measured over an interval of time. It is the proportion of persons with a particular disease or attribute at any time during the interval.

In particular, I think the above definitions clarify our confusion about how to compute, say, the period prevalence of diarrhea over one year. Namely, since prevalence is supposed to be a proportion of the population, each person should get counted at most once in the numerator, even if they develop diarrhea multiple times. So the calculation would be:

_period prevalence of diarrhea_
= (_number of people who had diarrhea at any point during the year_) / (_total population_)
= (_number of people who either had diarrhea at the beginning of the year or developed it at least once during the year_) / (_total population_)

However, the above CDC definitions remain ambiguous about what should go in the denominator, i.e. what is meant by "total population."  Is it the population at the beginning of the time period?  Or the average population over the time period?  Or something else?

Example C under Figure 3.1 on the [CDC site](https://www.cdc.gov/csels/dsepd/ss1978/lesson3/section2.html) uses the population at the beginning of the year, even though some of the people died, so the average population would be smaller.

Seems like in the context of a cohort study, maybe you'd be interested in the population at the beginning; whereas if you're talking about the population of a country, which changes continuously with birth, death, and migration, the average population would be more appropriate.

Thoughts, anyone?

Reply:
On second thought, maybe "total population" is not ambiguous -- should "total population" mean "all people who entered your study population at any point during the time period," and the numerator would be a subset of these people?

Reply:
What I have seen in practice is researchers work with a random sample of people and calculate the ratio from this sample.  I guess this implies that the "total population" they are generalizing to from the sample is the population at the *end* of the period.  But maybe they then assume this is not too different from the beginning or middle of the period.

Reply:
in the diarrhea literature: 

Diarrhoea incidence | 
The number of diarrhoea episodes per person-time (incidence density) or over a defined period of time (cumulative incidence).

Diarrhea point prevalence | 
The proportion of the population experiencing a diarrhoea day at the time of interest, e.g. the day of a surveillance visit or the day before.

Diarrhoea period prevalence | 
The proportion of the population experiencing at least 1 day with diarrhoea over a pre-defined time window (recall period) prior to a given point in time, e.g. a surveillance visit by the study team.

Reply:
I think to get a 'prevalence' of a disease you would do a cross-sectional survey. if you asked the questions- did you have a diarrhea day yesterday and then tallied all the people who said yes over the number of people sampled, you would get a point-prevalence estimate for the population.

Reply:
corrected "the number of current cases," to % of persons with the thing. thanks for catching it!